docs/docs/20-usage/75-project-settings.md

```diff
@@ -41,11 +41,19 @@ Only server admins can set this option. If you are not a server admin this optio
 
 ## Custom trusted clone plugins
 
-The clone step may require Git credentials (e.g. for private repos) which are injected via `netrc`.
+During the clone process, Git credentials (e.g., for private repositories) may be required.
+These credentials are provided via [`netrc`](https://everything.curl.dev/usingcurl/netrc.html).
 
-They are only injected into trusted plugins listed in the env var `WOODPECKER_PLUGINS_TRUSTED_CLONE` or in this repo setting.
+These credentials are injected only into trusted plugins specified in the (admin) environment variable `WOODPECKER_PLUGINS_TRUSTED_CLONE` or declared in this repository-level setting.
 
-This allows you to use a trusted plugin for in the clone section or as a step to pull or push using your git credentials.
+With these credentials, it’s possible to perform any Git operations, including pushing changes back to the repo.
+To prevent unauthorized access or misuse, a plugin whitelist is required—either on the instance level or the repository level.
```

Comment:
Please dont use the term "whitelist" anymore:

```suggestion
To prevent unauthorized access or misuse, a plugin allowlist is required, either on the instance level or the repository level.
```